.github/tekton/goreleaser.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: goreleaser
+  labels:
+    app.kubernetes.io/version: "0.2"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/categories: Automation, Publishing
+    tekton.dev/tags: golang, release-automation, package
+    tekton.dev/displayName: "GoReleaser"
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  description: |-
+    GoReleaser builds Go binaries for several platforms.
+    It creates a GitHub release and then pushes a Homebrew formula to a tap repository.
+  params:
+    - description: base package to build in
+      name: package
+      type: string
+    - default: bot-token-github
+      description: name of the secret holding the github-token
```

Comment:
I prefer to let all description starts with a capital letter.

Reply:
Please see https://hub.tekton.dev/tekton/task/goreleaser.

Reply:
Some of your descriptions start with a capital letter, some are not. A good practice is keeping the same style.

Reply:
I'd prefer to remain those descriptions unchanged. Because they are mainly copied from Tekton catalog.